lte/gateway/python/magma/pipelined/tests/test_encoding.py

```diff
@@ -13,12 +13,13 @@
 
 import unittest
 
```

Comment:
The tests in this file are snapshot tests -- can you add a note, perhaps just here at the top of the file, detailing how you created the snapshot

Basically just want to assure future readers that the snapshot was created with third-party tools, rather than by our own code -- for correctness purposes